src/fetch_reply.py

```
@@ -934,71 +1077,341 @@ def _process_single_email(self, msg):
-                    # Create draft as usual
-                    draft_id = self.graph_client.create_threaded_reply_draft(
-                        message_id, reply_text, source_account
-                    )
-                    
-                    if draft_id:
-                        logger.info(f"Threaded draft saved: {draft_id}")
-                        draft_created = True
-                    else:
-                        logger.warning(f"Threaded draft save failed")
+                        else:
+                            logger.warning(f"Threaded draft save failed")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)